src/components/policy/policy_regular/src/policy_table/enums.cc

```diff
@@ -302,107 +302,107 @@ bool EnumFromJsonString(const std::string& literal, Parameter* result) {
   }
 }
 
-bool IsValidEnum(AppHMIType val) {
-  switch (val) {
-    case AHT_DEFAULT:
-      return true;
-    case AHT_COMMUNICATION:
-      return true;
-    case AHT_MEDIA:
-      return true;
-    case AHT_MESSAGING:
-      return true;
-    case AHT_NAVIGATION:
-      return true;
-    case AHT_INFORMATION:
-      return true;
-    case AHT_SOCIAL:
-      return true;
-    case AHT_BACKGROUND_PROCESS:
-      return true;
-    case AHT_TESTING:
-      return true;
-    case AHT_SYSTEM:
-      return true;
-    case AHT_PROJECTION:
-      return true;
-    case AHT_REMOTE_CONTROL:
-      return true;
-    default:
-      return false;
-  }
-}
-const char* EnumToJsonString(AppHMIType val) {
-  switch (val) {
-    case AHT_DEFAULT:
-      return "DEFAULT";
-    case AHT_COMMUNICATION:
-      return "COMMUNICATION";
-    case AHT_MEDIA:
-      return "MEDIA";
-    case AHT_MESSAGING:
-      return "MESSAGING";
-    case AHT_NAVIGATION:
-      return "NAVIGATION";
-    case AHT_INFORMATION:
-      return "INFORMATION";
-    case AHT_SOCIAL:
-      return "SOCIAL";
-    case AHT_BACKGROUND_PROCESS:
-      return "BACKGROUND_PROCESS";
-    case AHT_TESTING:
-      return "TESTING";
-    case AHT_SYSTEM:
-      return "SYSTEM";
-    case AHT_PROJECTION:
-      return "PROJECTION";
-    case AHT_REMOTE_CONTROL:
-      return "REMOTE_CONTROL";
-    default:
-      return "";
-  }
-}
-bool EnumFromJsonString(const std::string& literal, AppHMIType* result) {
-  if ("DEFAULT" == literal) {
-    *result = AHT_DEFAULT;
-    return true;
-  } else if ("COMMUNICATION" == literal) {
-    *result = AHT_COMMUNICATION;
-    return true;
-  } else if ("MEDIA" == literal) {
-    *result = AHT_MEDIA;
-    return true;
-  } else if ("MESSAGING" == literal) {
-    *result = AHT_MESSAGING;
-    return true;
-  } else if ("NAVIGATION" == literal) {
-    *result = AHT_NAVIGATION;
-    return true;
-  } else if ("INFORMATION" == literal) {
-    *result = AHT_INFORMATION;
-    return true;
-  } else if ("SOCIAL" == literal) {
-    *result = AHT_SOCIAL;
-    return true;
-  } else if ("BACKGROUND_PROCESS" == literal) {
-    *result = AHT_BACKGROUND_PROCESS;
-    return true;
-  } else if ("TESTING" == literal) {
-    *result = AHT_TESTING;
-    return true;
-  } else if ("SYSTEM" == literal) {
-    *result = AHT_SYSTEM;
-    return true;
-  } else if ("PROJECTION" == literal) {
-    *result = AHT_PROJECTION;
-    return true;
-  } else if ("REMOTE_CONTROL" == literal) {
-    *result = AHT_REMOTE_CONTROL;
-    return true;
-  } else {
-    return false;
-  }
-}
+// bool IsValidEnum(AppHMIType val) {
```

Comment:
why keep these commented code?

Reply:
@yang1070 Fixed in d1308970ca2afad7de251f6ef744d94c4422cb42